resources-generator/build.gradle.kts

```diff
@@ -40,18 +48,76 @@ dependencies {
     detektPlugins(Deps.Libs.Jvm.detektFormatting)
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+    withJavadocJar()
+    withSourcesJar()
+}
+
 publishing {
-    repositories.maven("https://api.bintray.com/maven/icerockdev/plugins/moko-resources-generator/;publish=1") {
+    repositories.maven("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/") {
         name = "bintray"
 
         credentials {
-            username = System.getProperty("BINTRAY_USER")
-            password = System.getProperty("BINTRAY_KEY")
+            username = System.getenv("OSSRH_USER")
+            password = System.getenv("OSSRH_KEY")
         }
     }
     publications {
-        register("maven", MavenPublication::class) {
+        register("mavenJava", MavenPublication::class) {
```

Comment:
go try `publications.withType<MavenPublication> {` instead this. if it will be as i think - we will be able to remove `from(components["java"])` and just will use `publish` task for all